packages/commerce-sdk-react/src/hooks/ShopperContexts/cache.ts

```diff
@@ -5,16 +5,32 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 import {ApiClients, CacheUpdateMatrix} from '../types'
+import {getShopperContext} from './queryKeyHelpers'
 
 type Client = ApiClients['shopperContexts']
 
-/** Logs a warning to console (on startup) and returns nothing (method is unimplemented). */
-const TODO = (method: keyof Client) => {
-    console.warn(`Cache logic for '${method}' is not yet implemented.`)
-    return undefined
-}
 export const cacheUpdateMatrix: CacheUpdateMatrix<Client> = {
-    updateShopperContext: TODO('updateShopperContext'),
-    createShopperContext: TODO('createShopperContext'),
-    deleteShopperContext: TODO('deleteShopperContext')
+    createShopperContext(customerId, {parameters}) {
+        return {
+            invalidate: [{queryKey: getShopperContext.queryKey(parameters)}]
+        }
+    },
+    updateShopperContext(_customerId, {parameters}) {
+        return {
```

Comment:
Do we want to add a TODO as a reminder that we need to invalidate all the cache in case data stored in cache is affected by the changes to the context?

Reply:
I added a TODO in a comment with a link to the ticket for the work. I chose not to use the TODO method since this work is follow up work and not a developer reminder before going GA